server/controllers/addItem.js

```diff
@@ -0,0 +1,34 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+const base = new Airtable({ apiKey: Airtable_API_KEY }).base('appAZnpLnWP0wjAc6');
```

Comment:
What is the string inside the brackets in the following line:

```
.base('appAZnpLnWP0wjAc6')

```

Reply:
base code
Airtable provide you with this, it's like a key for your workspace and does not matter if showen

Reply:
cool c: